project/app/main.py

```diff
@@ -1,26 +1,63 @@
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
+from fastapi.responses import Response, JSONResponse
+from fastapi.requests import Request
+from os import getenv
 import uvicorn
 
 from app.api import predict, viz
 
 app = FastAPI(
-    title='YOUR-PRODUCT-TEAM-NAME DS API',
-    description='Replace this placeholder text',
-    version='0.1',
-    docs_url='/',
+    title="Labs26-StorySquad-DS-Team B",
+    description="A RESTful API for the Story Squad Project",
+    version="0.1",
+    docs_url="/",
 )
 
+
+@app.middleware("http")
+async def check_auth_header(request: Request, next_call):
+    """
+    Function acts as a custom middlewear for http requests, checks if the
+    request header contains Authorization and if the value is equal to the
+    expected value that is stored in memory.
+    Arguments
+    -----------
+    `request`: {fastapi.requests.Request} - requesting client
+    `next_call`: {typing.Callable} - callable function that the middlewear
+    is intercepting
+    Returns
+    ---------
+    `response`: {fastapi.responses.Response} - response from calling the
+    function
+    """
+    bad_response = Response(
+        status_code=403, content="PATH FORBBIDEN", media_type="text/html"
```

Comment:
Typo? Should it be `"PATH FORBIDDEN"`

Reply:
fixed with commit: 8a0aa51d7fa53455362125f7e54aafca6499821d

Reply:
Not quite fixed -- missing one pesky 'D' in FORBIDDEN